subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/DependencySubstitutionInternal.java

```diff
@@ -21,12 +21,14 @@
 import org.gradle.api.artifacts.result.ComponentSelectionDescriptor;
 import org.gradle.api.internal.artifacts.ivyservice.resolveengine.result.ComponentSelectionDescriptorInternal;
 
+import java.util.List;
+
 public interface DependencySubstitutionInternal extends DependencySubstitution {
     void useTarget(Object notation, ComponentSelectionDescriptor selectionDescription);
 
     ComponentSelector getTarget();
 
-    ComponentSelectionDescriptorInternal getSelectionDescription();
+    List<ComponentSelectionDescriptorInternal> getSelectionDescription();
```

Comment:
Could this be renamed to `getSelectionDescriptions()`?

Reply:
Fixed in 8da6fa1550045e2a42f82945d8abffbfa32f1ce9